global/prometheus-alertmanager-operated/values.yaml

```diff
@@ -28,13 +28,6 @@ slack:
   vmware: {}
 
 # Values for PagerDuty receivers.
-pagerduty:
```

Comment:
I see `.Values.pagerduty.*` still being used in various places, for instance here: https://github.com/sapcc/helm-charts/blob/2db664cd2ac9e95cfc293a52750567e634139912/global/prometheus-alertmanager-operated/templates/_alertmanager.yaml.tpl#L759

Reply:
Don't get it. 
Of course `.Values.pagerduty_sap.` is still in :: just `.Values.pagerduty.` is removed (and  I can't find any anymore). 
Do I miss something?

Reply:
I though I saw 2 occurrences of the old `.Values.pagerduty.`. Nevermind they seem to be gone